aeon/dj_pipeline/report.py

```diff
@@ -468,24 +468,24 @@ class VisitSummaryPlot(dj.Computed):
     def make(self, key):
         from aeon.dj_pipeline.utils.plotting import (
             plot_foraging_bouts,
-            plot_summary_per_visit,
+            plot_visit_daily_summary,
         )
```

Comment:
Should we move these imports to the beginning of the module? Are there reasons for these to be imported only within this `make()`? (avoiding circular dependency?)

Reply:
It's just cuz they are util functions to be used only within that method..  circular dependency wasn't an issue.